docs/VnVPlan/VnVPlan.tex

```diff
@@ -209,55 +209,488 @@ \section{System Test Description}
+Initial State: User finds submits that same item for cheap price from location X.
+
+Input: User submits receipt containing items that are expensive from location Y.
+          
+Output: System provides recommendation to purchase item from location X.
+
+Test Case Derivation: System must be aware of the users purchasing activity and recommend the user with cheaper alternative locations for the same item.
+          
+How test will be performed: Tester will submit items purchased for a cheap price on one user account (A) and then submit the same item for an expensive amount from a different store with another user account (B). Tester verifies that user B receives a recommendation from the system to go to the same place as user A.
+
+\item{\textbf{FRT-FR9-2}}
+
+Control: Manual, Dynamic
+          
+Initial State: User finds submits that same item for expensive price from location X.
+
+Input: User submits receipt containing items that are cheap from location Y.
+          
+Output: System provides recommendation to purchase item from location Y.
+
+Test Case Derivation: System must be aware of the users purchasing activity and recommend the user with cheaper alternative locations for the same item.
+          
+How test will be performed: Tester will submit items purchased for an expensive price on one user account (A) and then submit the same item for a cheap amount from a different store with another user account (B). Tester verifies that user A receives a recommendation from the system to go to the same place as user B.
+
+\end{enumerate}
+
+\paragraph{Grocery Items Interface}
+
+These tests pertain to all the functional requirements related to the systems grocery item interface.
+
+\begin{enumerate}
+
+\item{\textbf{FRT-FR10-1}}
+
+Control: Manual, Dynamic
+          
+Initial State: User has no spending data or submitted items.
+
+Input: User submits a receipt.
+          
+Output: System provides an interface showing all purchased items along with price, location, and date-time.
+
+Test Case Derivation: System must provide the user with all of the data that the user has provided the system such that the user can review their own purchasing data.
+          
+How test will be performed: Tester will submit a receipt and then verify to see that the items have been added in the purchases interface.
+
+\item{\textbf{FRT-FR10-2}}
+
+Control: Manual, Dynamic
+          
+Initial State: User has existing spending data and submitted items.
+
+Input: User submits a receipt.
+          
+Output: System provides an interface showing all purchased items along with price, location, and date-time.
+
+Test Case Derivation: System must provide the user with all of the data that the user has provided the system such that the user can review their own purchasing data.
+          
+How test will be performed: Tester will submit a receipt and then verify to see that the items have been added in the purchases interface along with all of the preexisting items .
```

Comment:
There's a space at the end of the sentence between the last word and the paragraph.